### PR TITLE
Feature/us21036 author template update

### PR DESCRIPTION
**Pull request by laurasprauer**
## Problem
[Rally Task](https://rally1.rallydev.com/#/66096747656d/custom/24109743995?detail=%2Fuserstory%2F602986759080)

## Corresponding Branch
https://github.com/crdschurch/crds-components/pull/1304

## Testing
- Does not exactly match the [Invision](https://projects.invisionapp.com/share/WY10HCY0MKX4#/screens) designs - uses pre-existing components that are very similar with minimal differences, need to check if this is ok or if I need to implement a "theme" for certain crds-components **I talked to Becky and it's currently ok to keep as-is with the current component designs**
- There was a request for a "Learn More" link for authors with long bios, [this](https://www.crossroads.net/media/authors/baylee-waterman) is the longest author bio I could find and it seems to look alright - is a "Learn More"/Expand button still needed? **I talked with Becky and no expand is needed for the bio**
- The "Latest" tab should display the first 2 most recent articles and then in the side bar the next most recent articles, messages, podcasts, or videos, if there are no articles, it will show the 2 most recent messages, podcasts, or videos instead.

### CODE REVIEW

File: _layouts/author.html [hunk 1]
```diff
@@ -4,60 +4,198 @@
...
+        {% assign tabs_array = tabs_array | push: "messages" %}
+      {% endif %}
+
+      {% if podcasts.size > 0 %}
+        {% assign tabs_array = tabs_array | push: "podcasts" %}
+      {% endif %}
+
+      {% assign latest_array = articles | concat: videos | concat: messages | concat: podcasts | sort: 'date' | reverse %}
+
+      <crds-tabs tabs='{{tabs_array | jsonify }}' mobile-dropdown navigation-class="align-tabs-left">
```
Comment (silasbaur) [inline _layouts/author.html:43]: I like what you did here!

**Review by silasbaur — CHANGES_REQUESTED** [review]
A couple of inline comments. Let me know if you want to talk through anything
1. This image is not rendering correctly something is wrong with imgix optimizer here. Also this image is way too tall. Should have a height of 410px
Screen Shot 2021-07-09 at 9 59 15 AM
2. I would not include podcast here. Just include episodes and other media. Podcasts will not show a duration and it will look weird.
Screen Shot 2021-07-09 at 10 01 12 AM
3. I fixed the crds-media-object image. `data-optimize-img` does not work on components very well. Once the image component is fixed we should not have problems anymore. I just added
4. I also fixed the author image. When you first load the page, the image was small and then became bigger. I fixed this by adding a class of `full-width`.
5. These images are messed up here. I would take a look at where else we have cards similar to this and copy what they are doing.
Screen Shot 2021-07-09 at 10 05 26 AM

File: _layouts/author.html [hunk 2]
```diff
@@ -4,60 +4,198 @@
 
 {% assign author = site.authors | where: 'full_name', page.title | first %}
```
[the remark this answers is not in the document]
Comment (laurasprauer) [reply _layouts/author.html:5]: I think I need to keep this as is - page.author is not defined

**Comment by laurasprauer** [comment]
@silasbaur fixes are up!

I'm not sure how to fix the article heroes height issue however (number 1 in your list)
I pulled the code from the hero on the articles pages - is there a better place to pull this code from? or did you want me to set a hard height on these images?

**Review by dholloran-ample — CHANGES_REQUESTED** [review]
@laurasprauer 

- Left 2 inline comments
- For the image issue I wonder if we need to use `<crds-image />` for the images. Have you tried loading them outside of the `<crds-tabs />` component? I wonder if it is related to something stencil does?

**Comment by laurasprauer** [comment]
@dholloran-ample I pushed up all your fixes - I added a max-height to the image issue, apparently this was a style included on the articles page (where I copied this code from) that I had missed, it should look the same now! However it's still "fuzzy" while the one on the individual article page is not, even tho they are essentially using the same code? but it looks like all the images are "fuzzy" - is this just because so many are getting loaded at once and it's a dev env?

**Review by dholloran-ample — APPROVED** [review]

**Pull request merged**